src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/Android/ButtonExtensions.Android.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using AndroidX.AppCompat.Widget;
+
+namespace Xamarin.Platform
+{
+	public static class ButtonExtensions
+	{
+		public static void UpdateColor(this AppCompatButton appCompatButton, IButton button)
+		{
```

Comment:
are we ready to discuss style ? if so, we're moving to c#8, I propose that we move to expression-body methods for those extensions. It really sends the message "don't bother, it's just a one-liner boilerplate helper"

Reply:
Converted these to expression-body